### ISSUE TITLE
Blockchain Api Key

### ISSUE THREAD

**Issue by CrazyTapok-bit** [blockchain/service-my-wallet-v3#536]
Hi. I am trying to get an API key to work with the Blockchain. I have already applied for several times using the link https://www.blockchain.com/api/request. the first time it was 10.09. 10 days have passed, there is no answer. What should I do to start using it?

**Comment by Yommyjay222** [comment]
I can help you out

**Comment by blondeaboutlife** [comment]
Yeah, same issue here - what is going on??? Do they just not deal/close these issues? Did you ever get a key?

**Comment by Yommyjay222** [comment]
Hi how can I help you can you text me privately I should guide you and assist you 

                                                                       /bot

**Comment by CrazyTapok-bit** [comment]
> Yeah, same issue here - what is going on??? Do they just not deal/close these issues? Did you ever get a key?

I don't know what's going on with them, but I never got a response(

**Comment by Yommyjay222** [comment]
Paste your number so I can chat you privately

**Comment by blondeaboutlife** [comment]
Probably about to go the way of gocoin :-/ I stalked one of lead developers (via his commits) and sent him an email - also no reply

**Comment by blondeaboutlife** [comment]
> Paste your number so I can chat you privately
That is just not sensible now is it?

**Comment by Yommyjay222** [comment]
Text me your email or number

**Comment by blondeaboutlife** [comment]
> Text me your email or number

If you were able to help, you would help in public.

**Comment by Yommyjay222** [comment]
I would help you personally

**Comment by CrazyTapok-bit** [comment]
All this is very suspicious. Why can't you help in public like @blondeaboutlife says

**Comment by blondeaboutlife** [comment]
I've reported him/her/it and blocked him/her/it
